### DISCUSSION TITLE
No actions for device using philips tap dial

### DISCUSSION THREAD

**Discussion by Bobbb-afk** [PiotrMachowski/Home-Assistant-custom-components-SmartThings-Soundbar#74] [Q&A]
Hi Team,

I'm fairly new to HA so pardon me if this is a stupid question. My soundbar is working great via HA (Samsung Q990B) but I'm trying to setup a Philips tap dial for some basic functions using this blueprint https://github.com/Ddemooij1984/Home-assistent/blob/2faa4942270ffb1975cbc0b64e7499aa4ff65ce9/philips_tap_dial_switch%20volume-and-actions.yaml

Unfortunately I have no actions to assign to the tap dial as specified within the media_player.py file. Any easy way to add these functions so I can assign them to the tap dail?

Thanks!

**Comment by PiotrMachowski** [comment]
You just have to use media_player actions instead of device actions
https://www.home-assistant.io/integrations/media_player
